### PR TITLE
Feature/healthchecks

### PR DESCRIPTION
**Pull request by nlx-sascha**

### CODE REVIEW

File: containers/php/fpm/php-fpm-healthcheck [hunk 1]
```diff
@@ -0,0 +1,139 @@
+#!/bin/sh
```
Comment (liwo) [inline containers/php/fpm/php-fpm-healthcheck:1]: Wie bekommen wir mit, dass es zu diesem Script Updates gibt und wie sieht der Updateprozess aus?

**Review by liwo — CHANGES_REQUESTED** [review]

File: containers/php/fpm/php-fpm-healthcheck (hunk 1 above)
Comment (nlx-sascha) [reply containers/php/fpm/php-fpm-healthcheck:1]: The update process is manual. The last release is from 2020 and the last commit happened 10 months ago. The source of the script is mentioned in the comments. I think we don't have to worry about the update.

File: containers/proxy/config/sites-enabled/health-check.conf [hunk 2]
```diff
@@ -0,0 +1,20 @@
+server {
+    listen 90 default_server;
+
+    location /nginx_status {
```
Comment (nlx-lars) [inline containers/proxy/config/sites-enabled/health-check.conf:4]: Where or how is that location used?

**Review by nlx-lars — APPROVED** [review]

File: containers/proxy/config/sites-enabled/health-check.conf (hunk 2 above)
Comment (nlx-sascha) [reply containers/proxy/config/sites-enabled/health-check.conf:4]: This is for the exporter

**Pull request merged**